seed/challenges/getting-started.json

```diff
@@ -140,7 +140,7 @@
         [
           "http://i.imgur.com/FkEzbto.gif",
           "A gif showing how you can click your profile image in your upper right hand corner to your code portfolio and connect GitHub.",
-          "Check out your code portfolio. Click your picture your upper right hand corner. To activate your code portfolio, you'll need to link your GitHub account with Free Code Camp.",
+          "Check out your code portfolio. Click your picture in your upper right hand corner. To activate your code portfolio, you'll need to link your GitHub account with Free Code Camp.",
```

Comment:
We are moving fast right now so we didn't see your comment. I definitely agree that is better. Want to form a PR?